common/resource/params.go

```diff
@@ -66,5 +69,7 @@ type (
 		ArchiverProvider         provider.ArchiverProvider
 		Authorizer               authorization.Authorizer // NOTE: this can be nil. If nil, AccessControlledHandlerImpl will initiate one with config.Authorization
 		AuthorizationConfig      config.Authorization     // NOTE: empty(default) struct will get a authorization.NoopAuthorizer
+		isolationGroupState      isolationgroup.State
```

Comment:
field should be exported

Reply:
Could you also make change in `cmd/server.go`? If we do that, such error can be detected early.

Reply:
Sure, let me do that